frontend/cypress/pages/metrics/metrics.ts

```diff
@@ -205,6 +213,19 @@ class Metrics {
     this.progressBar().should('be.visible')
     this.progressBar().should('not.exist')
   }
+
+  checkRequiredFields() {
+    this.cycleTimeSettingDoneColumnToNone()
+    this.nextButton().should('be.disabled')
+    this.cycleTimeSettingDone()
+    this.checkRealDone()
+    this.nextButton().should('be.enabled')
+
+    this.classificationClear().click({ force: true })
+    this.nextButton().should('be.disabled')
+    this.checkClassification()
+    this.nextButton().should('be.enabled')
+  }
```

Comment:
Move those e2e check under `e2e` folder, please study POM, the related links are in onboarding mural.